cmd/ocm-load-test.go

```diff
@@ -130,15 +138,18 @@ func run(cmd *cobra.Command, args []string) error {
 		time.Duration(viper.GetInt("duration"))*time.Minute,
 		vegetaRate,
 		connection,
-		viper.Sub("tests")); err != nil {
-		return fmt.Errorf("running load test: %v", err)
+		viper.Sub("tests"),
+		logger,
+		cmd.Context()); err != nil {
+		logger.Fatal(cmd.Context(), "running load test: %v", err)
```

Comment:
It may be Github's diff viewer but the indentation looks a little weird here.

Reply:
It's not Github, it is go directly beacuse of the multiline initialization and the short hand if, it defaults to that position. This is already running the go fmt :shrug:

Reply:
haha so interesting. I will keep that in mind for the future, thanks for sharing that info!